RFS/build.gradle

```diff
@@ -47,6 +47,8 @@ dependencies {
     implementation group: 'software.amazon.awssdk', name: 's3'
     implementation group: 'software.amazon.awssdk', name: 's3-transfer-manager'
     implementation group: 'software.amazon.awssdk.crt', name: 'aws-crt'
+    implementation group: 'software.amazon.awssdk', name: 'auth'
+    implementation group: 'software.amazon.awssdk', name: 'sdk-core'
```

Comment:
Shouldn't this be picked up by the new library?

Reply:
Yes!  Please don't require downstream consumers to know what other jars they'll need to depend upon.  You can also expose those details as an api dependency if there's some AWS context, etc that needs to interface with the rest of the exposed implementation.

Reply:
Removed